### PR TITLE
feat(de-translations): added de-DE translations to the UI

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Type:** feat

**Description:**

follow up of https://github.com/verdaccio/ui/pull/432

image

### CODE REVIEW

**Comment by juanpicado** [comment]
LGTM, but I'd rather trust on @DanielRuf 👍

File: i18n/translations/de-AT.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
+{
+    "copy-to-clipboard": "In die Zwischenablage kopieren",
+    "author-anonymous": "Anonymus",
+    "action-bar-action": {
+        "visit-home-page": "Auf die Homepage gehen",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "visit-home-page": "Zur Homepage",
```

File: i18n/translations/de-AT.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        "open-an-issue": "Einen Fehler melden",
+        "download-tarball": "Tarball Herunterladen"
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "download-tarball": "Tarball herunterladen"
```

I'm not quite sure if it's usual to say "tarball" in German, you could also just say `Archiv herunterladen`.

File: i18n/translations/de-AT.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        "documentation": "Dokumentation",
+        "registry-info": "Registrierungsinformationen",
+        "greetings": "Hallo "
+    },
+    "search": {
+        "packages": "Pakete suchen"
+    },
+    "auto-complete": {
+        "loading": "wird geladen...",
+        "no-results-found": "kein Ergebnis gefunden"
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "no-results-found": "Kein Ergebnis gefunden"
```

File: i18n/translations/de-AT.json [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    },
+    "versions": {
+        "current-tags": "Aktuelle Tags",
+        "version-history": "Versionsgeschichte",
+        "not-available": "Nicht verfügbar"
+    },
+    "package": {
+        "published-on": "Veröffentlicht am {{time}} •",
+        "version": "v{{version}}",
+        "visit-home-page": "Auf die Homepage gehen",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "visit-home-page": "Zur Homepage",
```

File: i18n/translations/de-AT.json [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+        "username": "Benutzername",
+        "password": "Passwort"
+    },
+    "form-placeholder": {
+        "username": "Dein Benutzername",
+        "password": "Dein sicheres Passwort"
+    },
+    "form-validation": {
+        "required-field": "Dieses Feld ist erforderlich",
+        "required-min-length": "Dieses Feld erfordert die Mindestlänge von {{length}}",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "required-min-length": "Dieses Feld erfordert eine Mindestlänge von {{length}}",
```

oder 

```suggestion
        "required-min-length": "Dieses Feld erfordert mindestens {{length}} Zeichen",
```

File: i18n/translations/de-AT.json [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+        "unable-to-sign-in": "Anmeldung nicht möglich",
+        "username-or-password-cant-be-empty": "Benutzername und Passwort dürfen nicht leer sein!"
+    },
+    "help": {
+        "title": "Noch kein Paket publiziert.",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "title": "Noch kein Paket veröffentlicht.",
```

File: i18n/translations/de-AT.json [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+        "sub-title": "Um dein erstes Paket einfach zu publizieren:",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "sub-title": "Um dein erstes Paket einfach zu veröffentlichen:",
```

File: i18n/translations/de-AT.json [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+        "second-step-command-line": "npm publish --registry {{registryUrl}}",
+        "third-step": "3. Diese Seite aktualisieren."
+    },
+    "sidebar": {
+        "detail": {
+            "latest-version": "Letzte v{{version}}",
+            "version": "v{{version}}"
+        },
+        "installation": {
+            "title": "Installierung",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
            "title": "Installation",
```

File: i18n/translations/de-AT.json [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+            "title": "Autor(in)"
+        },
+        "distribution": {
+            "title": "Neueste Distribution",
+            "license": "Lizenz",
+            "size": "Größe",
+            "file-count": "Anzahl der Dateien"
+        },
+        "maintainers": {
+            "title": "Betreuerinnen"
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
            "title": "Maintainer"
```

In IT it's rather usual to talk about "maintainers" even in German.

File: i18n/translations/de-AT.json [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+        },
+        "contributors": {
+            "title": "Beitragenden"
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
            "title": "Contributor"
```

File: i18n/translations/de-AT.json [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+        },
+        "engines": {
+            "npm-version": "NPM Version",
+            "node-js": "NODE JS"
+        }
+    },
+    "footer": {
+        "powered-by": "Angetrieben von",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]: This is rather for engines :)

```suggestion
        "powered-by": "powered by",
```

or

```suggestion
        "powered-by": "Unterstützt von",
```

File: i18n/translations/de-AT.json [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+        "made-with-love-on": "Gemacht mit <0>♥</0> in"
+    },
+    "button": {
+        "close": "Schließen",
+        "cancel": "Beenden",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "cancel": "Abbrechen",
```

File: i18n/translations/de-AT.json [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+        "login": "Einloggen",
+        "logout": "Ausloggen",
+        "go-to-the-home-page": "Auf die Homepage gehen",
```
Comment (jan-auer) [inline i18n/translations/de-AT.json]:
```suggestion
        "go-to-the-home-page": "Zur Homepage",
```

**Comment by priscilawebdev** [comment]
@jan-auer Thank you very much for all the suggestions 😉 I've applied all of them.

File: tools/webpack.dev.config.babel.js [hunk 14]
```diff
@@ -38,7 +38,6 @@ export default {
     new HTMLWebpackPlugin({
       __UI_OPTIONS: JSON.stringify({
         base: new URL('/', 'https://localhost:4872'),
-        language: 'es-ES',
```
Comment (juanpicado) [inline tools/webpack.dev.config.babel.js]: 👍

File: src/App/load-dayjs-locale.ts [hunk 15]
```diff
@@ -32,6 +32,18 @@ function loadDayJSLocale() {
         dayjs.locale('pt-br');
       }
       break;
+    case 'de':
```
Comment (juanpicado) [inline src/App/load-dayjs-locale.ts]: would not be `de-de`?

Comment (priscilawebdev) [reply src/App/load-dayjs-locale.ts]: nope... de is the default one 😉

**Pull request merged**